storage/src/mmr/mem.rs

```diff
@@ -31,7 +31,7 @@ pub struct Mmr<H: CHasher> {
 
     // The position of the oldest element still retained by the MMR, or the size of the MMR if there
```

Comment:
nit: I think there is probably a clearer way to word this comment.

Maybe something like `The position of the oldest element still retained by the MMR. If there are no retained nodes (or the MMR is empty), this is equal to the size of the MMR.`

The part that confuses me is the `position of the oldest element still retained b ythe MMR, or the size of the MMR` (makes it sound like you are storing different values here depending on what is going on).

Reply:
How about "The highest position for which this MMR has been pruned, or 0 if this MMR has never been pruned, or 0 if the MMR has never been pruned."

Reply:
Works for me 👍